libs/vm-modules/src/core/structured_data.cpp

```diff
@@ -192,7 +192,9 @@ void StructuredData::Bind(Module &module)
       .CreateMemberFunction("set", &StructuredData::SetArray<double>)
       .CreateMemberFunction("set", &StructuredData::SetArray<fixed_point::fp32_t>)
       .CreateMemberFunction("set", &StructuredData::SetArray<fixed_point::fp64_t>)
-      // .CreateMemberFunction("set", &StructuredData::SetArray<Fixed128>)
+      .CreateMemberFunction("set", &StructuredData::SetObjectArray<Fixed128>)
+      .CreateMemberFunction("set", &StructuredData::SetObjectArray<String>)
+      // .CreateMemberFunction("set", &StructuredData::SetObjectArray<UInt256Wrapper>)
```

Comment:
needs a TODO(issue number if you're not going to remove this comment)
Same goes for line 185 on this file

Reply:
added